cypress/support/ListPage.js

```diff
@@ -21,9 +21,9 @@ export default url => ({
         logout: '.logout',
         bulkActionsToolbar: '[data-test=bulk-actions-toolbar]',
         customBulkActionsButton:
-            '[data-test=bulk-actions-toolbar] button:first-child',
+            '[data-test=bulk-actions-toolbar] button[aria-label="Reset views"]',
```

Comment:
:+1: Better but it would even better to not rely on those css selectors at all, just as users do. This is something we should fix in a pr dedicated to e2e tests though